Library/Homebrew/migrator.rb

```diff
@@ -55,7 +55,10 @@ def initialize(formula)
 
     @old_tabs = oldpath.subdirs.each.map { |d| Tab.for_keg(Keg.new(d)) }
     @old_tap = old_tabs.first.tap
-    raise MigratorDifferentTapsError.new(formula, old_tap) unless from_same_taps?
+
+    unless ARGV.force? || from_same_taps?
```

Comment:
Mind using an `if` here instead; I always find `unless` with multiple arguments hard to parse mentally.